app/signals/apps/signals/tokens/token_generator.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2023 Gemeente Amsterdam
+from secrets import token_urlsafe
+
+
+class TokenGenerator:
+    """This class should be used whenever a secure token of some sort needs
+    to be generated within the application.
+    """
+
+    def __call__(self) -> str:
+        """Generate a secure token.
+
+        Returns
+        -------
+        str
+            The generated secure token.
+        """
+        return token_urlsafe()
```

Comment:
Isn't it a bit too much to create a TokenGenerator class? We can also use the `token_urlsafe` function where needed (or one of the other functions, like `token_hex`)?